_data/chains/eip155-17000.json

```diff
@@ -8,7 +8,8 @@
   ],
   "faucets": [
     "https://faucet.holesky.ethpandaops.io",
-    "https://holesky-faucet.pk910.de"
+    "https://holesky-faucet.pk910.de",
+    "https://chat.blockscan.com/index?a=0x035219a183f9e31514e0d7c23a85e3e76d7bf167"
```

Comment:
this does not look like a faucet to me

Reply:
This is a chat faucet, similar to the ones on Discord 

https://x.com/etherscan/status/1707712285115056469?s=46

Reply:
Do let me know if this is not suitable, will remove it from the PR!

Reply:
bit on the fence here - when I open this page I just get to some start-page

Reply:
Reechek